consensus/propagation/have_wants.go

```diff
@@ -104,14 +103,9 @@ func ReqLimit(partsCount int) int {
 }
 
 func (blockProp *Reactor) requestFromPeer(ps *PeerState) {
+	blockProp.waitForFirstProposal()
```

Comment:
why do we need to wait for the first proposal? if we remove this does the loop still work?